PRINCIPLES.md

```diff
@@ -10,22 +10,22 @@ These principles were derived from modern software operations but are rooted in
 
 ## Principles
 
-1. **The principle of declarative desired state**
+1. **The system's desired state is described declaratively**
 
     A system managed by GitOps must have its _Desired State_ expressed declaratively as data in a format writable and readable by both humans and machines.
 
-2. **The principle of immutable desired state versions**
+2. **State declarations are stored as immutable versions**
 
     _Desired State_ is stored in a way that supports versioning, immutability of versions, and retains a complete version history.
 
-3. **The principle of continuous state reconciliation**
+3. **State reconciliation is automated and continuous**
 
-    Software agents continuously, and automatically, compare a system's _Actual State_ to its _Desired State_.
+    Software agents continuously, and automatically, compare a system's state to its _Desired State_.
     If the actual and desired states differ for any reason, automated actions to reconcile them are initiated.
 
-4. **The principle of operations through declaration**
+4. **Intentional change happens only through changing state declarations**
```

Comment:
```suggestion
4. **System Changes are Only Through Version Control**
```

Reply:
```suggestion
4. **Changes are Only Through Version Control**
```

Reply:
🤏

Reply:
Hm, my gut response to "Changes are Versioned" for principle 4 is this crosses the line from brevity to loosing the intended meaning 🤔 🧠 

I think the main point of 4 is that – in principle – for a system to be considered GitOps-managed it must only use new versions of desired state declarations for all intentional changes to the system, not as one of many methods to change the system. Right?

There are exceptions noted - such as the example of "break glass" for manual intervention during incident management etc – during which time, this GitOps principle is considered to be temporarily suspended.

WDYT?

Reply:
"Updates to desired state are versioned"

Reply:
@moshloop suggested:

```suggestion
4. **Intentional Changes Only Through Versioning**
```